addons/linecompass/XEH_preInit.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+ADDON = false;
+#include "XEH_PREP.hpp"
+
+GVAR(drawEh) = -1;
```

Comment:
could wrap basically every except the last few lines into an if and check if the machine has an interface